catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -445,6 +463,19 @@
                 </configuration>
             </plugin>
             <plugin>
+                <groupId>org.codehaus.gmavenplus</groupId>
+                <artifactId>gmavenplus-plugin</artifactId>
+                <version>1.4</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>compile</goal>
```

Comment:
I don't think you need this goal since only your tests are written in groovy, the `testCompile` goal should handle those.